app/src/main/java/org/mozilla/fenix/experiments/NimbusFeatures.kt

```diff
@@ -116,4 +122,51 @@ class NimbusFeatures(private val context: Context) {
             return homeScreenFeatures[HomeScreenSection.topSites] == true
         }
     }
+
+    /**
+     * Component that indicates whether search term groups are active.
+     */
+    class SearchTermGroupFeatures(private val context: Context) {
+        /**
+         * This enum accompanies the `FeatureId.SEARCH_TERM_GROUPS` feature.
+         *
+         * These names should match the names of entries in the JSON.
+         */
+        @Suppress("EnumNaming")
+        private enum class SearchTermGroups(val default: Boolean) {
+            /**
+             * This covers search term groupings in the tabs tray, homescreen, and
+             * the tab settings.
+             */
+            searchTermGroups(true);
```

Comment:
@eliserichards Drive-by: You will want this to be turned off (set to `false`) by default. Similar to what @Amejia481 did for our 94 branch where we turn on/off home screen sections for the experiment (and default to false): https://github.com/mozilla-mobile/fenix/blob/releases_v94.0.0/app/src/main/java/org/mozilla/fenix/experiments/NimbusFeatures.kt#L54

(In 95, and on main, we default to true because the experiment will be over at that point).

Otherwise all users will see tab groups right away and for those users in control the groups would suddenly disappear when the user is enrolled (this happens async.) Also, all users who opted out of studies would also see tab groups.

So it's better to default to false. Then users in the experiment/treatment group will start seeing groups when they get enrolled, preventing the two problems above.

Reply:
I agree with @csadilek, as users will have a better experience with the suggested approach.